packages/fiori/src/DynamicPageTitle.ts

```diff
@@ -132,6 +135,22 @@ class DynamicPageTitle extends UI5Element {
 	@slot({ type: HTMLElement })
 	snappedHeading!: HTMLElement[];
 
+	/**
+	 * Defines the content of the snapped title on mobile devices.
+	 *
+	 * This slot is displayed only when the `DynamicPageTitle` is in the snapped state on mobile devices.
+	 * It should be used to provide a simplified, single-line title that takes up less space on smaller screens.
+	 *
+	 * **Note:**
+	 * - The content set in this slot **overrides** all other content set in the `DynamicPageTitle` slots when displayed.
+	 * - The slot is intended for a single `ui5-title` component.
+	 *
+	 * @public
+	 * @since 2.3.0
+	 */
+	@slot({ type: Title })
```

Comment:
You can read about developing slots here and their possible types here:
https://github.com/SAP/ui5-webcomponents/blob/main/docs/4-development/04-slots.md